__tests__/plugins_phones.test.js

```diff
@@ -53,18 +53,12 @@ describe("phones_plugin", () => {
     });
     describe("grabHrefTel", () => {
         it("should return a number", () => {
-            const line = "href='tel:+44 1632 960983'";
+            const line = `href="tel:+44 1632 960983"`;
             const plugin = new PhonesPlugin();
 
-            expect(plugin.grabHrefTel(line)).toEqual(
-                line.split("tel:")[1].split("'")[0].replace(/\s/g, "")
-            );
-        });
-        it("should not return bad data", () => {
```

Comment:
I think we should keep this unit test

don't trust people not to fuck up

What happens in the scenario of 

'xxx / xxx / xxx'